addons/account/wizard/account_payment_register_views.xml

```diff
@@ -55,7 +55,7 @@
                         </group>
                         <group name="group3"
                                attrs="{'invisible': ['|', ('payment_difference', '=', 0.0), '|', ('can_edit_wizard', '=', False), '&amp;', ('can_group_payments', '=', True), ('group_payment', '=', False)]}"
-                               groups="account.group_account_readonly">
+                               groups="account.group_account_invoice">
```

Comment:
You don't even need that group since it is needed by the main model of the form view